pkg/workspace/plugins.go

```diff
@@ -289,9 +289,20 @@ func (info PluginInfo) Install(tarball io.ReadCloser) error {
 	// If two calls to `plugin install` for the same plugin are racing, the second one will be unable to rename
 	// the directory. That's OK, just ignore the error. The temp directory created as part of the install will be
 	// cleaned up when we exit by the defer above.
+	fmt.Print("Moving plugin...")
```

Comment:
Where these `fmt.Printt`'s just for debugging, should we remove them if so?

Reply:
They were intentional because now the renaming takes quite a long time, the progress bar is at 100%, and it's totally not clear whether the command is stuck or not. The prints give some indication of the progress.
Let me know if there is a better way.

Reply:
> The prints give some indication of the progress.

Gotcha!